src/sentry/api/endpoints/org_auth_tokens.py

```diff
@@ -66,6 +66,11 @@ def post(
         if not name:
             return Response({"detail": ["The name cannot be blank."]}, status=400)
 
+        if len(name) > 255:
```

Comment:
I would suggest to use a shared constant or eval function, since this can diverge pretty easily.

Reply:
I changed this to reference a constant!